### PR TITLE
Update design using adidas YARN Design System

### PR DESCRIPTION
**Pull request by moelders**
Update adidas YARN Design System.

### CODE REVIEW

File: config/build.js [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
-    new webpack.ProvidePlugin({
-      '$': 'jquery',
-      'jQuery': 'jquery',
-      'window.jQuery': 'jquery'
-    }),
-    new CompressionWebpackPlugin()
-  ],
+  plugins: [ new CompressionWebpackPlugin() ],
   extend(config, { isDev }) {
-    if (!isDev) {
```
Comment (RecuencoJones) [inline config/build.js]: Revert this change!

**Review by RecuencoJones — CHANGES_REQUESTED** [review]

File: config/build.js (hunk 1 above)
Comment (moelders) [reply config/build.js]: > Revert this change!

Ouch!

**Review by RecuencoJones — APPROVED** [review]
LGTM! 🚀
